include/photogeo.h

```diff
@@ -93,10 +93,13 @@ struct ptg_tracing_parameters {
 /// Results from the tracing step.
 struct ptg_tracing_results {
     /// The outlines.
-    ptg_outline outlines;
+    ptg_outline** outlines;
```

Comment:
Reverse the order of these variables. I think this would make it easier to follow.